src/components/header/export-modal/renderer.tsx

```diff
@@ -80,7 +89,15 @@ class ExportModal extends React.PureComponent<Props, State> {
     const content = this.state.downloadVegaJSON ? this.props.vegaSpec : this.props.vegaLiteSpec;
     const filename = this.state.downloadVegaJSON ? `visualization.vg.json` : `visualization.vl.json`;
 
-    const blob = new Blob([JSON.stringify(content, null, 2)], { type: `application/json` });
+    if (this.state.includeConfig) {
+      const Newcontent = mergeDeep({ config: this.props.config }, content as any);
```

Comment:
Variables shouldn't start with capital letters. 

Why didn't the code I suggested not work?

Reply:
1. Sure I will change the name of the variable.
2. The second argument takes a priority , so If is pass custom config as first arguement and content (spec.config) as second arguement , the result is perfect , but the config (key) appears on top of the json.
That is why I have deleted the key and added it again , so is appears at the bottom

Reply:
I see. Can you add a comment to explain that?

Reply:
Sure!

Reply:
I don't see the comment. Can you only respond to my comments once you finished the task? It makes it easier for me to track what has happened and what still needs to be done. Okay?

Reply:
Okay! pushed changes. Should I make a branch for you to test?